collections/ansible_collections/cloudkit/templates/roles/ocp_virt_vm/tasks/delete.yaml

```
@@ -0,0 +1,73 @@
+---
+- name: Set VM facts
+  ansible.builtin.set_fact:
+    vm_name: "{{ template_parameters.vm_name }}"
+
+- name: Stop VirtualMachine
+  kubernetes.core.k8s:
+    api_version: kubevirt.io/v1
+    kind: VirtualMachine
+    name: "{{ vm_name }}"
+    namespace: "{{ vm_target_namespace }}"
+    state: present
+    definition:
+      spec:
+        running: false
+  ignore_errors: true
```

I don't think we should ignore errors on any of these tasks :thinking:

---

The idea here was to remove the VM even if it can't gracefully shutdown the VM.  I will remove it.